test/integration/manifests/swiftv2/long-running-cluster/pod.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: {{ .PodName }}
+  namespace: {{ .Namespace }}
+  labels:
+    kubernetes.azure.com/pod-network-instance: {{ .PNIName }}
+    kubernetes.azure.com/pod-network: {{ .PNName }}
+spec:
+  nodeSelector:
+    kubernetes.io/hostname: {{ .NodeName }}
+  containers:
+  - name: net-debugger
+    image: {{ .Image }}
+    command: ["/bin/bash", "-c"]
+    args:
+      - |
+        echo "Pod Network Diagnostics started on $(hostname)";
+        echo "Pod IP: $(hostname -i)";
+        echo "Starting HTTP server on port 8080";
+        
+        # Create a simple HTTP server directory
```

Comment:
do we need to define this index page instead of using a library image? please mention the use case if we are adding details to print